lib/db/set_test.go

```diff
@@ -997,6 +997,31 @@ func TestMoveGlobalBack(t *testing.T) {
 	}
 }
 
+func TestIssue5007(t *testing.T) {
```

Comment:
Add a description here of what this test actually tests for, please. I've realized it's super annoying in the future to run across an anonymous numbered test case.